contents/blog/posthog-vs-fullstory.md

```diff
@@ -0,0 +1,576 @@
+
+#### PostHog is an all-in-one product improvement platform
+While FullStory includes a few additional features, it is mostly focused on being a session replay tool, while PostHog has a much wider scope. That’s because PostHog is intended to be an all-in-one platform and can easily replace an entire stack of traditionally disparate tools, such as LaunchDarkly, Amplitude, and FullStory.  
+
+#### PostHog is for engineers, technical users, _builders_
+PostHog has been designed from the ground up to meet the needs of developers and product-focused engineers. This is reflected not only in the scope of the features, but also the documentation and platform architecture. For example, PostHog offers advanced tools for debugging user issues within session replays. 
+
+#### PostHog is open source
+As an open source platform, PostHog is almost entirely transparent. You can check out the PostHog repo to monitor code changes, build your own features and apps, and even interact with the team directly. FullStory, on the other hand, is entirely closed source.
+
+<ArrayCTA /> 
+
+### PostHog and FullStory pricing
+Both PostHog and FullStory offer three main pricing tiers, but how these tiers differ is actually quite different. 
+
+**PostHog** has a simple free tier with a limited feature set and a generous allowance of 1 million free events and 15,000 free sessions each month, after which users move to a more full featured paid plan which is billed based on usage. The third tier is an enterprise add-on, which expands the feature set further with the permissioning and privacy tools needed by large organizations.
+
+>  Check [PostHog’s pricing page](/pricing) to find out more about what’s included in each tier, or to see how per-event pricing scales.
+
+**FullStory**, on the other hand, offers only three premium, paid-for tiers, which differ in available features. The basic Business plan offers minimal features and a limited 14-day trial, while the following Advanced and Enterprise plans increase the feature scope and, presumably, cost. 
+
+We say ‘presumably’ because FullStory’s pricing isn’t transparent and there’s no way to know exactly how much each tier costs. At PostHog, that’s [an approach we strongly disagree with](/blog/transparent-enterprise-pricing). 
+
+In order to offer a straightforward comparison of PostHog and FullStory, we’ll focus this article on the _enterprise_ versions of both products — that is, the most feature-complete tiers. In this way we can clearly establish the features which both products do or do not have without getting bogged down in what features are available on each tier. 
+
+> If a particular feature is critically important to you, we strongly advise checking the details of FullStory’s pricing, as not all features are equally available and pricing is not transparent. [Unlike PostHog's](/pricing). 
```

Comment:
Unless there's a way to condense this a lot, I think we should move this into the FAQ. These guides should get into features as fast as possible and this is a lot wordage to get through.